fe/src/test/java/org/apache/doris/alter/AlterTest.java

```diff
@@ -135,10 +135,10 @@ public void testConflictAlterOperations() throws Exception {
         waitSchemaChangeJobDone(true);
         
         // enable dynamic partition
+        // not adding the `start` property so that it won't drop the origin partition p1, p2 and p3
```

Comment:
Do we need a add another test case to cover the TTL(auto drop out of date partition) feature？

Reply:
OK